bool LedState
bool LaserState
bool GripperState
bool PeltierState
bool Stepper1Fault
bool Stepper2Fault
